include/tvm/relay/adt.h

```diff
@@ -241,14 +241,18 @@ class MatchNode : public ExprNode {
   /*! \brief The match node clauses. */
   tvm::Array<Clause> clauses;
 
+  /*! \brief Is the match complete (cover all cases)? */
```

Comment:
Small nitpick: I think the comment here and in `adt.py` should be a little clearer about what the variable means: It means whether the match _should_ be complete, not whether it _is_ complete (which the typechecker checks).

```suggestion
  /*! \brief Should this match be complete (cover all cases)? 
   *  If yes, the type checker will warn about any missing cases.
   */
```